packages/protocol/contracts/common/UsingRegistry.sol

```diff
@@ -42,7 +42,7 @@ contract UsingRegistry is Ownable {
     abi.encodePacked("FeeCurrencyWhitelist")
   );
   bytes32 constant FREEZER_REGISTRY_ID = keccak256(abi.encodePacked("Freezer"));
-  bytes32 constant GOLD_TOKEN_REGISTRY_ID = keccak256(abi.encodePacked("GoldToken"));
+  bytes32 constant GOLD_TOKEN_REGISTRY_ID = keccak256(abi.encodePacked("CeloToken"));
```

Comment:
Changing the REGISTRY_ID will require a governance proposal to additionally add the new registry ID, pointing to the existing CeloToken (i.e. GoldToken) proxy.